### PR TITLE
Add blogpost about smart home day 2019

### PR DESCRIPTION
**Pull request by Confectrician**
Feedback welcome.

Slides are depending on openhabfoundation/openhabfoundation.github.io#67.

Signed-off-by: Jerome Luckenbach <github@luckenba.ch>

### CODE REVIEW

**Comment by Confectrician** [comment]
Maybe something to know before giving feedback:
I've got some pictures and will check them and add one or too before finishing this pr.

**Comment by ghys** [comment]
@Confectrician do you have/need access to the CRM?

**Comment by Confectrician** [comment]
have: no
need: idk, does it bring significant benefits?

**Comment by ghys** [comment]
Well it can upload images and add blog posts with all the required metadata, but ultimately it only helps making PRs here, so as you already know how to do it the "regular" way it's okay too ;)

File: blog/2019-07-18-smart-home-day-darmstadt-2019.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+| When the Smart Home goes Public. Research on the Usage of openHAB in a University Building | Myriam Guedey, HfT Stuttgart                   | [Slides](https://www.openhabfoundation.org/documents/2019-07_SHD2019_Smarthome_goes_Public.pdf) |
+| Optimisation of in-house consumption of solar energy with openHAB                          | Kai Kreuzer, openHAB Foundation e.V.           | [Slides](https://www.openhabfoundation.org/documents/2019-07_SHD2019_Solar_Energy_Optimisation.pdf) |
+| SmaC: openHAB in a smart caravan                                                           | Marius Falk, Hahn-Schickard-Gesellschaft e.V.  | [Slides](https://www.openhabfoundation.org/documents/2019-07_SHD2019_Smart_Caravan.pdf) |
+| A new frontend for openHAB                                                                 | Yannick Schaus, openHAB Foundation e.V.        | [Slides](https://www.openhabfoundation.org/documents/2019-07_SHD2019_A_new_frontend_for_openHAB.pdf) |
+| openHAB Alexa: Past, Present and Future (Dan Cunningham, openHAB Foundation e.V.)          |                                                |                                                               |
```
Comment (kaikreuzer) [inline blog/2019-07-18-smart-home-day-darmstadt-2019.md]:
```suggestion
| openHAB Alexa: Past, Present and Future  | Dan Cunningham, openHAB Foundation e.V.          |                                                |                                                               |
```

**Comment by Confectrician** [comment]
>Is this still WIP or do you consider it complete?

I am not pleased with the header pic.
I will do some small changes. After that it's complete from my side.

If i am right, Dan has sent his slides meanwhile. So we can add them too before publishing this one.

**Comment by Confectrician** [comment]
Ok, looks better now.
I have also added Dans slides and filed a pr at the openhabfoundation website.

**Pull request merged**